airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -3242,6 +3242,15 @@ definitions:
           - "$ref": "#/definitions/IterableDecoder"
           - "$ref": "#/definitions/XmlDecoder"
           - "$ref": "#/definitions/ZipfileDecoder"
+      lazy_read_pointer:
```

Comment:
I really like the fact that this is decoupled from the Python implementation as adding a retriever has caused development in the Connector Builder in the past ❤️